src/main/java/lt/andriaus/hangman/database/Database.java

```diff
@@ -4,6 +4,8 @@
 import java.util.Optional;
 
 public interface Database<E> {
+    Optional<E> loadRandom();
```

Comment:
kam toks metodas reikalingas?

Reply:
kai kuriu naują žaidimą, man reikia gaut random žodį, ir aš naudoju tą patį duombazės interfacą tiek žaidimams, tiek žodžiams. Žodžiu, šitas metodas suranda man random žodį, https://github.com/Andrius-ir-Andrius/hangman/pull/5#discussion_r571916645 čia va šitą sprendžiu